src/chapters/aspdotnet-model-validation/validation-controller.rst

```diff
@@ -0,0 +1,100 @@
+.. _validating-models:
+
+Validating Models in a Controller
+=================================
+
+Validation involves both model and controller components of an MVC application.
+After we have defined validation rules using attributes on the model, we must also update the controller to ensure that the rules are checked and appropriate action is taken when validation fails.
+
+Validation Flow
+---------------
+
+Before diving into the details of the code, let's consider the logical flow of control for validating data in a request.
+Recall our ``POST`` action method from the previous chapter, which used model binding to create new ``Event`` objects from form submissions.
+
+.. sourcecode:: csharp
```

Comment:
Since we have updated this code to use a ViewModel, I suggest using the updated version as the motivating example. This will make the transition into the next section easier to follow.

Reply:
Updated to explain that this is the old flow and with a viewmodel, there would be a different flow!